README.md

```diff
@@ -59,6 +59,68 @@ ua =
   |> UAParser.parse()
 ```
 
+## Example
+There is an example Phoenix application with a simple endpoint/route to retrieve the user agent header.
+To run it, simply cd into the `example` directory, and run the following:
+
+```shell
+$ mix do deps.get, compile
+$ npm i && brunch build
+# change the configuration to match your own postgres username/password
+$ mix do ecto.create, ecto.migrate
+$ iex -S mix phoenix.server
+# visit localhost:4000/user_agent
+```
+
+It can also be used as a simple plug:
+
+```shell
+mix new ua_plug
+cd ua_plug
+```
+
+And now set up `mix.exs`:
+
+```elixir
+# ...
+	defp deps do
+		[
+     {:cowboy, "~> 1.0.0"},
+     {:plug, "~> 1.0"},
+     {:ua_parser, git: "https://github.com/doomspork/ua_parser.git",
```

Comment:
I thought pointing to master would just make it easier to keep the example app working with the repository in master because then we can test live changes w/o changing the example app code and still have the latest rather than just what was last published for a version.